apps/web/test/web/controllers/opportunities_controller_test.exs

```diff
@@ -0,0 +1,22 @@
+defmodule Web.OpportunitiesControllerTest do
+  use Web.ConnCase
+
+  import Data.Factory
```

Comment:
Since you elected to use the `Opportunities` classes to handle inserts, you don't need this factory import.

The factories are part of https://github.com/thoughtbot/ex_machina